db/migrations/02_create_tables.sql

```diff
@@ -1,5 +1,7 @@
 CREATE TABLE users (id SERIAL PRIMARY KEY, username VARCHAR(60) UNIQUE, password VARCHAR(60), email VARCHAR(70) UNIQUE, phone_number INTEGER);
 CREATE TABLE spaces (id SERIAL PRIMARY KEY, user_id INTEGER REFERENCES users(id), price INTEGER, name VARCHAR(60), description VARCHAR(240));
-CREATE TABLE bookings (id SERIAL PRIMARY KEY, user_id INTEGER REFERENCES users(id), space_id INTEGER REFERENCES spaces(id), date VARCHAR(60), bookable? BOOLEAN);
+CREATE TABLE bookings (id SERIAL PRIMARY KEY, user_id INTEGER REFERENCES users(id), space_id INTEGER REFERENCES spaces(id), date VARCHAR(60), bookable BOOLEAN);
+
+ALTER TABLE bookings DROP COLUMN bookable;
```

Comment:
We may need to edit this table further (since bookings will need to be in a state of approved/disapproved requests), but will have to discuss this as a group